host/2.0/stretch/amd64/python.Dockerfile

```diff
@@ -20,8 +21,29 @@ RUN wget https://github.com/Azure/azure-functions-python-worker/archive/$WORKER_
 
 RUN cp -R /azure-functions-python-worker/python /azure-functions-host/workers/python
 
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 RUN apt-get update && \
-    apt-get install -y git make build-essential libssl-dev zlib1g-dev libbz2-dev \
+    apt-get install apt-transport-https
+
+# Support for MSSQL ODBC
+RUN apt-get update && apt-get install -my wget gnupg
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+RUN curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list > /etc/apt/sources.list.d/mssql-release.list
```

Comment:
the image is actually based on Debian 9 (stretch), not Ubuntu 16.04. I know they are similar, but we should probably be using `https://packages.microsoft.com/config/debian/9/prod.list` instead.

Reply:
Cool, I'll test that and commit